core/lib/dal/src/blocks_dal.rs

```diff
@@ -1611,6 +1724,7 @@ impl BlocksDal<'_, '_> {
         bootloader_hash: H256,
         default_aa_hash: H256,
         protocol_version_id: ProtocolVersionId,
+        is_rollup: bool,
```

Comment:
how will it work for `no-da-validium`? It's still not a rollup, but we don't expect anything in this other table do we?
Also we generally prefer not to have such flags (`is_rollup`/`is_validium`), but I don't see a quick way to improve it

Reply:
For `no-da-validium` the `da-dispatcher` will just put empty values into that separate table, so they are not null and the related batches can be queried for `commitBatches`, but it still follows the general flow. That will be supported on the contracts side, and if the `inclusion_data` is empty - it means that we don't need to do any onchain checks

Agree with the `is_rollup` point, but unfortunately I don't have a great solution for it too

Reply:
You may change it to `with_da_inclusion_info` to better indicate what does this flag mean, and also add a doc-comment like `/// When `with_da_inclusion_info` is `true`, only batches for which custom DA inclusion information has already been provided will be included.